Resources/Prototypes/Recipes/Construction/Graphs/structures/girder.yml

```diff
@@ -145,6 +145,22 @@
               amount: 2
               doAfter: 2
 
+        - to: shuttleInteriorWall
+          completed:
+            - !type:SnapToGrid
+              southRotation: true
```

Comment:
Is this needed because the girder SnapToGrid was removed?

Reply:
Honestly, that's just how they are all set up.